packages/ui/src/Radio.module.scss

```
@@ -0,0 +1,93 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.wrapper {
+  display: inline-flex;
+  // inline-flex changes vertical align compared to display: flex
+  // setting vertical-align allows us to easily use radio button
+  // inline e.g. next to the text
+  // Most common use case will be still the usage in a form where parent
+  // wrapper controls this.
+  vertical-align: bottom;
```

Should we also vertically center radio input, label and help icon?
```suggestion
  vertical-align: bottom;
  align-items: center;
```

---

Perfect!